### DISCUSSION TITLE
What about notes?

### DISCUSSION THREAD

**Discussion by angoca** [osmlab/osm-data-model#39] [General]
Does this GitHub project include discussions about notes? Or only map elements (node, way, relation, tags)?

**Comment by joto** [comment]
No, notes are outside the scope of this discussion here.

**Comment by angoca** [comment]
Ok. However, the project title is "OSM Data Model," and the definition on the [Readme](https://github.com/osmlab/osm-data-model/blob/ce76a15f0e6e4895387b5f6a094019a2cebeb120/README.md) is: "The OSM Data Model determines how our data is structured in the OSM database." These things imply the data on the whole database schema and notes are also part of this schema:

![OSM Data model](https://wiki.openstreetmap.org/w/images/thumb/5/58/OSM_DB_Schema_2016-12-13.svg/587px-OSM_DB_Schema_2016-12-13.svg.png)
[Image from the Wiki](https://wiki.openstreetmap.org/wiki/File:OSM_DB_Schema_2016-12-13.svg).

Probably, the title should be adjusted, and the Readme file should be more specific by specifying [OSM Elements](https://wiki.openstreetmap.org/wiki/Elements).

Finally, the result of these discussions will define the next database structure and API, and personally, I would like the notes to be included in this improvement.

**Reply by joto** [reply]
Okay, let me rephrase that: For me, when I talk about the data model, notes are not included. Here is the quote from my study:

_While writing this paper a lot of topics were suggested that it could address. But much of this is out of scope. This is about the OSM data model, the nodes, ways, relations, tags, changesets. It is not about anything user facing, it is not about tagging, it is not about the OSM website. The API is only in scope so far as it talks about those OSM data model issues. Other parts of the API, such as user
management, notes, GPX upload, etc. are not in scope. (That’s why this is paper is not called “Ideas for API 0.7” 1 or something like that). I agree that there are lots of other things in OSM which could be improved, but this paper is not about them._

I personally think things are complex enough that way and it makes sense to leave notes and many other things out here, but of course I can't and won't stop anybody from discussing anything and everything here.

**Comment by mmd-osm** [comment]
Please add your comments about Notes on the rails repo instead. Either create a new issue or add it to one of the existing ones like https://github.com/openstreetmap/openstreetmap-website/issues/3831

**Reply by mnalis** [reply]
- (ref: it seems it was opened in https://github.com/openstreetmap/openstreetmap-website/issues/3932)
